cap/modules/auth/utils.py

```diff
@@ -42,3 +43,32 @@ def _create_or_update_token(name, token):
     _token.expires_at = token.get('expires_at')
 
     return _token
+
+
+def get_oidc_token():
+    """Retrieve the token in order to use the CAP OIDC client to get info."""
+    resp = requests.post(
+        url=current_app.config.get('OIDC_TOKEN_API'),
+        headers={
+            'Content-Type': 'application/x-www-form-urlencoded'
+        },
+        data={
+            'client_id': current_app.config.get('CERN_CLIENT_ID'),
+            'client_secret': current_app.config.get('CERN_CLIENT_SECRET'),
+            'grant_type': 'client_credentials',
+            'audience': 'authorization-service-api'
```

Comment:
with new CERN OIDC updates, they should return with the token the user e-groups..lets handle from there